src/libraries/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/MemoryMappedFileSecurity.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
```

Comment:
This file is WIP.

Reply:
I filled this file, although I'm unsure if I need to add more method overrides. I'll continue comparing this with the existing derived classes we already have.

Reply:
Update: After deeper investigation, this should be all I need to include in this file.